lib/Runtime/Library/JSON.cpp

```diff
@@ -431,10 +431,16 @@ namespace JSON
 
         if (value == nullptr)
         {
-            Js::PropertyString* propertyString = Js::PropertyString::TryFromVar(key);
-            if (propertyString)
+            Js::JavascriptString * propertyKey = key;
```

Comment:
This is going to conflict with another JSON change that went in earlier today, might want to rebase your change and reapply if necessary